### ISSUE TITLE
Checkbox in table not styled

### ISSUE THREAD

**Issue by asztalosdani** [ColinDuquesnoy/QDarkStyleSheet#52]
Hi,
the checkboxes in the table are not styled.
![image](https://cloud.githubusercontent.com/assets/9664767/25223683/58610972-25bc-11e7-9a13-d532e9a43933.png)
My setup (just in case it matters):
Windows 10
Python 2.7
PyQt4
QDarkStyle 2.3

**Comment by ColinDuquesnoy** [comment]
Thank you for reporting. Can you provide a sample code that helps reproduce your issue?

**Comment by Raincode** [comment]
Hi,

I played around just trying to style it the way checkBoxes are styled in general, but I think the colors might not play so nicely with the selection background color: http://imgur.com/a/uDdpm

I just added a TableWidget in Designer and then I did
```
    QTableWidgetItem *item = new QTableWidgetItem("Test");
    item->setCheckState(Qt::Checked);
    ui->tableWidget->setItem(0, 0, item);
```

**Comment by ColinDuquesnoy** [comment]
@asztalosdani I think this issue cannot be solved with a stylesheet, you need to implement a custom delegate to customize items in a view: http://doc.qt.io/qt-4.8/qabstractitemdelegate.

@Raincode I have adjusted checkbox/radiobutton colors and view item selection/hover color to have a better contrast.

**Comment by ColinDuquesnoy** [comment]
Fixed in PR #67
